app/src/app.js

```diff
@@ -10,265 +10,21 @@
  */
 
 import * as React from 'react';
-import {useRef, useState} from 'react';
-
-import DeckGL from 'deck.gl';
-import {COORDINATE_SYSTEM} from '@deck.gl/core';
-import {IconLayer, PointCloudLayer} from '@deck.gl/layers';
-import ReactMapGL, {NavigationControl} from 'react-map-gl';
-
-import type {PickInfo} from "@deck.gl/core/lib/deck";
-
-const MAPBOX_TOKEN = process.env.MAPBOX_TOKEN;
-
-const MIN_ELEVATION = 10;
-const ICON_MAPPING = {
-  marker: {x: 0, y: 0, width: 128, height: 128, mask: true},
-};
-
-type ViewState = {
-  latitude: number,
-  longitude: number,
-  zoom: number,
-  bearing: number,
-  pitch: number,
-};
-
-type Point = {
-  latitude: number,
-  longitude: number,
-  height: number,
-  rssi: number,
-  bearing: number,
-  message: string,
-};
-
-type HoverInfo = PickInfo<Point>;
+import MapScreen from './screens/mapscreen';
```

Comment:
nit, should we make filenames match module names, like we do in our other repos? `MapScreen.js`